src/DisplayFormats/include/DisplayFormats/DisplayFormats.h

```diff
@@ -11,6 +11,10 @@ namespace orbit_display_formats {
 
 [[nodiscard]] std::string GetDisplaySize(uint64_t size_bytes);
 
+enum class TimeUnit { kNanosecond, kMicrosecod, kMillisecond, kSecond, kMinute, kHour, kDay };
+std::string DisplayTimeUnit(TimeUnit unit);
```

Comment:
I would suggest to call this function `GetDisplayTimeUnit` which makes it consistent with the other functions below.